src/test/unit/cli_unittest.cc

```diff
@@ -235,7 +235,7 @@ size_t getEEPROMStorageSize() {
 
 void setPrintfSerialPort(struct serialPort_s) {}
 
-static const box_t boxes[] = { { 0, "DUMMYBOX", 0 } };
+static const box_t boxes[] = { { "DUMMYBOX", 0, 0 } };
```

Comment:
what's the reason for changing this?

Reply:
If I have changed `box_t` order I need to change initializer order also.